index.js

```diff
@@ -36,7 +36,7 @@ async function getBalance(address, network) {
     try {
         let networkDetails = config.networks.testnet;
         if (network === 'main') networkDetails = config.networks.main;
-        const provider = new ProxyProvider(networkDetails.provider, timeOut);
+        const provider = new ProxyProvider(networkDetails.provider, {timeOut});
```

Comment:
@Minhaj-ul-abidin the object here should be `{timeout:timeOut}` . Can you please check this once.